data/hai/midas.txt

```diff
@@ -0,0 +1,940 @@
+			choice
+				`(Ignorance is bliss.)`
+				`(I prefer not to stay in the dark. Wake them.)`
+					goto light
+			label bliss
+			`You leave the traitor here on this world, and they seem thankful but deeply troubled by what happened. You wonder what could have attracted such attention to you. Whatever it was should probably be safely stashed away, or gotten rid off.`
+				accept
+			label light
+			`You throw a bucket of water on the crew member. They wake up, but at first refuse to tell you anything. Once you reveal you're aware of their marking, they concede to reveal something to you. They seem to recognize the hopelessness of their situation, and may be willing to bargain. "This sign means that I hold my life to someone, and have to do one task, any task, to repay them. There is something here, on this ship, of the highest importance to some... I may not, and will not say anything else, no matter what you try to make me do it. I am dead already, and they own me."`
+			`You nod, and they then proceed, "I will tell you the truth, but then you will let me go and we will never see each other again. But let me be clear, I will not speak, under any circumstances other than those."`
+			choice
+				`(This is it. I want to know all.)`
+				`(Ignorance is bliss.)`
+					goto bliss
+			`They take a deep breath. "An important organization wants these gold bars you have on board. They're not just normal gold. If you were to sell them to me, I could give you a very generous price of 1000000 credits per bar, and you wouldn't be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have every single one. I may have told you too much for them to allow me to live already."`
+			choice
+				`(Sell the bars.)`
+				`(Keep the bars, and drop that traitor on the planet.)`
+					goto bliss2
+				`(Keep the crew member for further interrogation.)`
+					goto escape
+			action
+				outfit "Gold Bars" -5
+				payment 5000000
+				set "midas: sold bars"
+			`The soon-to-be gone traitor nods. "It is the right choice you are making, Captain <last>. You will never be bothered by any of this again, that I promise. I will take these 5 piles, and any more will be sold little by little as you go on most worlds. If you sell them to any normal outfitters, we will easily get our hands on it, so it is no problem to us, but you would only be paid the price of brute gold for it."`
```

Comment:
`price of brute gold` sounds strange to me, also the description says that, too.

Reply:
Raw value of (pure) gold?

Reply:
Sure. I mostly just don't want them quoting the description.